src/api/users/tools/migrate/migrate.js

```diff
@@ -0,0 +1,68 @@
+const fetch = require('node-fetch');
```

Comment:
Should this live in `tools/` vs. `src/api/users/tools`?

Reply:
@manekenpix and I had some discussion about this too, we weren't sure so we decided to leave it here and see what others thought.

It originally was made to help with testing the paginated GET route, but now it instead dumps users to a json file so perhaps `tools/` is the better place after all.